lib/datadog/profiling/exporter.rb

```diff
@@ -52,7 +52,12 @@ def initialize(
 
       def flush
         worker_stats = @worker.stats_and_reset_not_thread_safe
-        start, finish, compressed_pprof = pprof_recorder.serialize
+        serialization_result = pprof_recorder.serialize
+        if serialization_result.nil?
+          Datadog.logger.debug('Skipped exporting profiling events due to failed serialization')
```

Comment:
Minor: Since this is debugging, I guess it's fine, but it does somewhat duplicate the debug message we already print in `StackRecorder#serialize`.